validator/project_validator.go

```diff
@@ -389,6 +404,108 @@ func dependencyCycleExists(node model.TVPair, allNodes []model.TVPair, visited m
 	return nil
 }
 
+func validatePcAliases(pc *model.ProjectConfig) ValidationErrors {
+	errs := []string{}
+	errs = append(errs, model.ValidateProjectAliases(pc.GitHubPRAliases, "GitHub PR Aliases")...)
+	errs = append(errs, model.ValidateProjectAliases(pc.GitHubChecksAliases, "Github Checks Aliases")...)
+	errs = append(errs, model.ValidateProjectAliases(pc.CommitQueueAliases, "Commit Queue Aliases")...)
+	errs = append(errs, model.ValidateProjectAliases(pc.PatchAliases, "Patch Aliases")...)
+	errs = append(errs, model.ValidateProjectAliases(pc.GitTagAliases, "Git Tag Aliases")...)
+
+	validationErrs := ValidationErrors{}
+	for _, errorMsg := range errs {
+		validationErrs = append(validationErrs, ValidationError{
+			Message: errorMsg,
+			Level:   Error,
+		})
+	}
+	return validationErrs
+}
+
+func validatePlugins(pc *model.ProjectConfig) ValidationErrors {
+	errs := ValidationErrors{}
+	webHook, _, err := model.IsWebhookConfigured(pc.Project, "")
```

Comment:
I wasn't totally sure if we should check against the project ref since it _would_ be an issue if a custom was configured on the project ref level and a BB config was specified in yaml if the BB config on the project page was empty.  This is because FindMergedProjectRef would merge the two entities and then both custom webhook and BB config would be defined.  However since this is really meant to strictly validate the project-config I think it's good to change this to use the passed in ProjectConfig.

Reply:
Sorry I think what I actually meant was that the ProjectConfig would be looked up when looking up the ProjectRef:
https://github.com/evergreen-ci/evergreen/blob/ccf58c1fb6813e43c1eb9b84006b31aab596bff0/model/project_ref.go#L2407-L2406
> This is because FindMergedProjectRef would merge the two entities and then both custom webhook and BB config would be defined.
It seems like when merging we should ignore both webhook & BB config from the project config if it's configured for the project ref (since that takes precedence) rather than erroring for validation.